manifests/operator-manifests/mattermost/operator.yaml

```diff
@@ -15,7 +15,7 @@ spec:
       serviceAccountName: mattermost-operator
       containers:
         - name: mattermost-operator
-          image: mattermost/mattermost-operator:v1.5.0
+          image: mattermost/mattermost-operator:v1.6.0
```

Comment:
@gabrieljackson did you double-check if the crds or any other manifest are up-to-date?

Reply:
Good point! It worked just fine, but I should update them just in case. Thanks for the reminder.

Reply:
@gabrieljackson let me know when you double-check